### PR TITLE
OAuth JWT Auth

### PR DESCRIPTION
**Pull request by Antanukas**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
         <lombok.version>1.16.8</lombok.version>
```
Comment (Antanukas) [inline pom.xml]: unused

File: src/main/app/auth/components/login/login_component.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+var module = require('main_module');
+
+function Controller($state, $stateParams, AuthService) {
```
Comment (zygisx) [inline src/main/app/auth/components/login/login_component.js]: `$stateParams` unused

File: src/main/app/auth/services/session.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+var module = require('main_module');
+
+function Service ($cookies, jwtHelper, $http) {
```
Comment (zygisx) [inline src/main/app/auth/services/session.js:3]: Why name not `Sessions`?

File: src/main/app/auth/components/login/login.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    border-top-right-radius: 0;
+  }
+
+  .alert {
+    margin-top: 15px;
+  }
+}
+
+
+
```
Comment (edmondas) [inline src/main/app/auth/components/login/login.scss]: useless empty lines :)

File: src/main/app/auth/components/login/login.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<form class="msi-signin">
+    <h2 class="msi-signin-heading">Please sign in</h2>
+    <label for="inputEmail" class="sr-only">Email address</label>
+    <input type="text" id="inputEmail"
+           class="form-control msi-input-email"
+           placeholder="Email address" required autofocus
+           ng-model="$ctrl.username">
```
Comment (edmondas) [inline src/main/app/auth/components/login/login.html:7]: maybe it's better to use data- prefix?

**Pull request merged**